contrib/masmx86/bld_ml32.bat

```diff
@@ -1,2 +1,2 @@
-ml /coff /Zi /c /Flmatch686.lst match686.asm
-ml /coff /Zi /c /Flinffas32.lst inffas32.asm
+ml /coff /Zi /c /safeseh /Flmatch686.lst match686.asm
+ml /coff /Zi /c /safeseh /Flinffas32.lst inffas32.asm

```

Comment:
Older ml.exe versions back to VS 2003.NET support /safeseh.  From MS: "Marks the object as either containing no exception handlers or containing exception handlers that are all declared with .SAFESEH"

So if the .asm files are build w/o SAFESEH, and the VS linker flag /SAFESEH:YES is set, then the build will fail.  The Linker flag YES is now the default in VS 2012+.  Building the .asm files with safeseh has no effect on earlier VS version where the linker is not defaulting to YES.

Having said all that, since there is a VS2015 specific asm build file being added, there is no danger in discarding this change if desired.  I could only test on VS2015 so I cannot validate the regression.